internal/tracinginterceptor/interceptor.go

```diff
@@ -177,12 +177,55 @@ func (i *Interceptor) CallOneway(ctx context.Context, req *transport.Request, ou
 
 // HandleStream implements interceptor.StreamInbound
 func (i *Interceptor) HandleStream(s *transport.ServerStream, h transport.StreamHandler) error {
-	panic("implement me")
+	parentSpanCtx, _ := i.tracer.Extract(i.propagationFormat, getPropagationCarrier(s.Request().Meta.Headers.Items(), i.transport))
+	span := i.tracer.StartSpan(
```

Comment:
Could we use the [same way](https://github.com/yarpc/yarpc-go/blob/31a6e88b74a5211b9a0f22b594d4e8fd7b50c801/transport/grpc/handler.go#L139-L146) to create a span? A few span tags (also `commonTracingTags`) are missing here. 

Same for the `CallStream`.